modules/nf-core/proteinortho/main.nf

```diff
@@ -0,0 +1,54 @@
+process PROTEINORTHO {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::proteinortho=6.3.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/proteinortho%3A6.3.0--h70414c8_0':
+        'quay.io/biocontainers/proteinortho:6.3.0--h70414c8_0' }"
+
+    input:
+    tuple val(meta), path(fasta_files, stageAs: "?/*")
+
+    output:
+    tuple val(meta), path("${meta.id}.proteinortho.tsv")                     , emit: orthologgroups
+    tuple val(meta), path("${meta.id}.proteinortho-graph")                   , emit: orthologgraph
+    tuple val(meta), path("${meta.id}.blast-graph")                          , emit: blastgraph
+    path "versions.yml"                                                      , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    proteinortho \\
+        $args \\
+        -cpus=$task.cpus \\
+        -project=$prefix \\
+        $fasta_files
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
```

Comment:
These parsing of versions seem very complicated. If they aren't present in the container (but are optional) then they just want to be blank lines in that case

Reply:
removed the "404 not found" output